src/Main.ts

```diff
@@ -274,13 +274,13 @@ export class Main {
         return `${(hs.media_url || hs.url)}/_matrix/media/r0/download/${mxcUrl.substring("mxc://".length)}`;
     }
 
-    public async getTeamDomainForMessage(message: any, teamId?: string) {
-        if (message.team_domain) {
+    public async getTeamDomainForMessage(message?: any, teamId?: string) {
```

Comment:
This feel like the wrong approach, why are we calling getTeamDomain on unknown messages?

Reply:
I should probably rename the method, it's more that we want to get the domain from the teamId.